# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295044395 28800
#      Fri Jan 14 14:33:15 2011 -0800
# Node ID 199543df2ece1587b51548a2226f51af1796a176
# Parent  06c80c90a2466ba904ba9aeef113296d6726662b
Delete unimplemented, now obsolete Matlab-compatability function flops.m

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-14  Rik  <octave@nomad.inbox5.com>
+
+	* miscellaneous/flops.m: Delete function which was unimplemented
+	in Octave and has been removed from Matlab since version 5.
+
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* specfun/nthroot.m: Add Seealso references to docstring.
 	
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* miscellaneous/module.mk: Remove texas_lotto.m from build system.
 
diff --git a/scripts/miscellaneous/flops.m b/scripts/miscellaneous/flops.m
deleted file mode 100644
--- a/scripts/miscellaneous/flops.m
+++ /dev/null
@@ -1,37 +0,0 @@
-## Copyright (C) 1996-2011 John W. Eaton
-##
-## This file is part of Octave.
-##
-## Octave is free software; you can redistribute it and/or modify it
-## under the terms of the GNU General Public License as published by
-## the Free Software Foundation; either version 3 of the License, or (at
-## your option) any later version.
-##
-## Octave is distributed in the hope that it will be useful, but
-## WITHOUT ANY WARRANTY; without even the implied warranty of
-## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-## General Public License for more details.
-##
-## You should have received a copy of the GNU General Public License
-## along with Octave; see the file COPYING.  If not, see
-## <http://www.gnu.org/licenses/>.
-
-## -*- texinfo -*-
-## @deftypefn {Function File} {} flops ()
-## This function is provided for @sc{matlab} compatibility, but it doesn't
-## actually do anything.
-## @end deftypefn
-
-## Author: jwe
-
-function retval = flops ()
-
-  if (nargin > 1)
-    print_usage ();
-  endif
-
-  warning ("flops is a flop, always returning zero");
-
-  retval = 0;
-
-endfunction
diff --git a/scripts/miscellaneous/module.mk b/scripts/miscellaneous/module.mk
--- a/scripts/miscellaneous/module.mk
+++ b/scripts/miscellaneous/module.mk
@@ -17,17 +17,16 @@ miscellaneous_FCN_FILES = \
   miscellaneous/debug.m \
   miscellaneous/delete.m \
   miscellaneous/dir.m \
   miscellaneous/dos.m \
   miscellaneous/dump_prefs.m \
   miscellaneous/edit.m \
   miscellaneous/fileattrib.m \
   miscellaneous/fileparts.m \
-  miscellaneous/flops.m \
   miscellaneous/fullfile.m \
   miscellaneous/getappdata.m \
   miscellaneous/getfield.m \
   miscellaneous/gunzip.m \
   miscellaneous/gzip.m \
   miscellaneous/info.m \
   miscellaneous/inputname.m \
   miscellaneous/isappdata.m \
